man/systemd.special.xml

```diff
@@ -285,15 +286,17 @@
         <term><filename>hibernate.target</filename></term>
         <listitem>
           <para>A special target unit for hibernating the system. This
-          pulls in <filename>sleep.target</filename>.</para>
+          pulls in <filename>sleep.target</filename> and
+          <filename>wakeup.target</filename>.</para>
```

Comment:
My patch is a copy-paste of the hooks used by sleep.target, as seen in diff context.

I'm not sure adding mentions of `systemd-hibernate.service` would make sleep.target any less confusing, with or without wakeup.target.

But I see an mistake where I described wakeup.target as "starting after suspend.target" etc.  That should have been worded consistent with sleep.target, but I left that as "A special target unit that is pulled in by suspend.target" etc.  I prefer the latter (with the looser, usage-focused wording which follows).  The former might not be correct (excuse this imprecision but I'm just about to go to sleep now :).